### PR TITLE
Get started page styling is off with small window sizes

### PR DESCRIPTION
**Pull request by laszlolm**
Solving #1977

### CODE REVIEW

File: app/style/GetStarted.less [hunk 1]
```diff
@@ -772,7 +772,7 @@
 
   .display-wallet.new {
     background-color: inherit;
-    margin-top: 60px;
+    /*margin-top: 60px;*/
```
Comment (matheusd) [inline app/style/GetStarted.less]: Remove.

Comment (laszlolm) [reply app/style/GetStarted.less]: Removed in latest commit

**Comment by laszlolm** [comment]
screen shot 2019-01-17 at 4 47 51 pm
screen shot 2019-01-17 at 4 47 58 pm
screen shot 2019-01-17 at 4 48 07 pm

Screenshots as requested @matheusd

**Comment by laszlolm** [comment]
Note: Any other components looking weird other than the buttons (looking at the blockchain sync bar being pushed to the left mainly) is out of the scope of this PR, I might address them later in another PR

**Pull request merged**